Solutions/Windows Security Events/Analytic Rules/TimeSeriesAnomaly-ProcessExecutions.yaml

```diff
@@ -64,5 +65,5 @@ entityMappings:
     fieldMappings:
```

Comment:
We should update account entities to strong identifiers

Reply:
Done